tox.ini

```diff
@@ -1,7 +1,8 @@
 [tox]
 minversion = 2.4
+envlist = py{2.7,3.6}
 
 [testenv]
+changedir = test
```

Comment:
Interesting point. I do agree that's a risk... and perhaps your technique is indeed superior. I do struggle a little bit with it for several reasons: (1) not all projects have a 'tests' directory, so to generalize the technique, one would need to create this directory. (2) pytest would only discover tests in this directory, meaning all tests would need to be moved to this directory, which isn't even possible for some tests like doctests and (3) When running tests against installed packages, they end up getting long, bulky paths in the traceback instead of the nearby local paths `fuzzy` vs `.tox/envname/lib/pythonN.N/site-packages/fuzzy` (maybe even absolute).